app/controllers/api/v1/rents_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
       end
 
       def create
-        if current_user.id != params[:rent][:user_id]
+        if params[:user_id].to_i != params[:rent][:user_id].to_i
           render json: { message: 'User id not allowed' }, status:	:unprocessable_entity
```

Comment:
You can put return `return render json ....` in only one line (if it is less than 100 chars)
Be careful with the tab between `status` and `unprocessable_entity`

Also in this case, the status code should be `forbidden` instead of `unprocessable_entity`, because there is no error in the creation or something, you are checking that an user creates a rent for someone else.
By the way, if you are thinking that the user is creating there own rents, we do not need to send the `user_id` over params, we have it in the `current_user`, so we can `merge` the parameters like here: https://stackoverflow.com/questions/16530532/rails-4-insert-attribute-into-params

We should discuss this later